### DISCUSSION TITLE
Where to find AS4 sending info

### DISCUSSION THREAD

**Discussion by TaliaEnj** [OxalisCommunity/oxalis#657] [General]
Hi all,
in order to setup correctly the OpenPeppol reporting documents, I need to read the transport protocol log or similar to collect the data,
as suggested in the Peppol Spech Tech: "All necessary raw data can be gathered by looking at the transport protocol and the contents 
of the Standard Business Document Header (SBDH) only".

Now, I'm able to collect data from oxalis.log file set into the logback.xml, but only for the OUBOUND traffic.

Where can I find the INBOUND info to collect? The receipt of the document received is not sufficient, and in the log I not find anything.

Maybe I need to activate a specific log for inbound documents?

Thanks a lot for any response.

**Comment by aaron-kumar** [comment]
@TaliaEnj : you can find answer of similar question at : https://norstellaoxalis.slack.com/archives/C02NS075S67/p1702711065367029 (norstellaoxalis.slack.com - Slack channel)

You can retrieved transport protocol from "inboundMetadata.getProtocol()"  for incoming message.

**Reply by lratt** [reply]
That's a private slack community

**Reply by dladlk** [reply]
The referenced message contains next text:

There are already guidelines mentioned Oxalis Wiki page : https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting-%E2%80%90-WIP - related to ExchangeTime - Message exchange time in UTC, precision up to second – Available in AS4 Transmission message “eb:UserMessage/eb:MessageInfo/eb:Timestamp”
In Oxalis, this can be retrieved from "inboundMetadata.getTimestamp()" for incoming message. For outbound message, you can retrieve it from "transmissionResponse.getTimestamp()". Make sure to truncate this timestamp up-to second.
Feel free to ask general Peppol questions. Oxalis Community will help you.

**Reply by lratt** [reply]
Thank you.

Who would I have to contact to get access to the slack?
Seems like there is a lot of useful discussion in there.

**Reply by aaron-kumar** [reply]
Yes @lratt  , https://norstellaoxalis.slack.com/ is Only available to Oxalis Paying members.  If you are Oxalis paying member, then send request with organization name to me or send request via https://www.oxalis.network/contact-us 
Further update is provided to reporting documentation: https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting
